package.json

```diff
@@ -30,7 +31,7 @@
     "build:icons:deprecated": "svgr --expand-props none --template src/utils/svgToIconTemplate/index.js -d src/icons/ src/svgs/deprecated/",
     "build:icons:index": "create-index src/icons/",
     "build:rollup": "rollup -c",
-    "build:babel": "babel src -d lib --extensions '.js,.ts,.tsx'",
+    "build:babel": "babel src -d lib --extensions '.js,.ts,.tsx' --ignore '**/*.test.{js,ts,tsx}'",
```

Comment:
We currently build test files, which seems unnecessary/inadvisable. 

I got side-tracked a bit about keeping actual source vs. test separate, could be a good Cooldown item to get things up to Open Source Best Practices.